### DISCUSSION TITLE
PAR2 and gocryptfs

### DISCUSSION THREAD

**Discussion by magnemann** [rfjakob/gocryptfs#624] [Q&A] [answered]
Hi,
Thanks for a great project.
Im doing a cold storage. 200GB. Encrypted with gocryptfs.
I will also make PAR2 files (multipar for windows) or the whole disk hopefulle to prevent some bitrot.
Do you recommend to make the PAR2 files on the encrypted files, or on the unencrypted files?

Thanks a lot, best regards Magne

**Comment by rfjakob** [comment, marked as answer]
I'm a bit late to the party, but encrypted files seems better.

Any single-bit-error on disk will translate to a complete 4kiB block lost in the unencrypted files. And I guess PAR will have a better chance of repairing single-bit-errors than completely lost blocks.
